misc/prost-codec/Cargo.toml

```diff
@@ -13,9 +13,6 @@ categories = ["asynchronous"]
 [dependencies]
 asynchronous-codec = { version = "0.6" }
 bytes = { version = "1" }
-prost = "0.11"
+protobuf = "3.2"
```

Comment:
This isn't really a `prost-codec` anymore, now that we are replacing the implementation with `protobuf` :sweat_smile: 

@mxinden Should we publish a new crate?

Reply:
:+1: